### PR TITLE
Revise Comms template

### LINKED ISSUE #4931: Revise Marketing Request template
Our [first revision](https://github.com/kubernetes/community/blob/master/.github/ISSUE_TEMPLATE/marketing-request.md) of the template is not quite the experience we want to provide our fellow Kubernetes contributors. Testing it with @parispittman, we found that: 

- Information fatigue happens fast. There is too much someone has to answer before they're ready to answer it.
- Follow up happens outside of GitHub. This leads to a gap in awareness of the status and next steps. 
- There are multiple possible 'next steps' after opening up. In some cases, a brainstorm session would be helpful, others need copy editing only, others still are ready to publish and looking to do so asap.

In response to to those challenges, we'd like to drastically simplify and encourage continuing the conversation in GitHub for visibility. 

Proposal: 

- Have a single, main question: What do you want to send out? (Please include a draft in Hackmd or Google Doc)
- Request next steps: Please @ the best person to help you next: 
  - X for copy editing or brainstorming
  - Y to send out the message asap

Open questions: 

1. is it better to have the X & Y be specific individuals or should we use our 'titles' in the working group and point people to that list? 
2. Is this triage the responsibility of leads or shadows? 

This issue can close when we have a new template that addresses the main point of simplifying the workflow and clarifying responsibility for next steps.

### PR DESCRIPTION
**Pull request by mbbroberg**
With a focus on ease of use and clarity around communication timeline. Also, we self-market ourselves, because we definitely should do that.

Signed-off-by: Matthew Broberg <matt@opensource.com>

**Which issue(s) this PR fixes**:

Fixes #4931 

/sig contribor-experience
/area contributor-comms

### CODE REVIEW

**Comment by mbbroberg** [comment]
@kaslin I'd love your take on this.

**Comment by kaslin** [comment]
Generally I think it looks good. The one thing that stands out to me is that if I was filling this out, I think it would be convenient to have a list of the possible comms types like from line 41 in the deleted stuff. I don't think we need to add that part back in though. It might be better to have something like "For an idea of the types of communications, check out the marketing team page (here)[https://github.com/kubernetes/community/tree/master/communication/marketing-team]." I think it would make sense to add a list of communication types that we commonly handle to the marketing page? This gives askers the freedom to just make their ask if they're already confident in it, or to find more info to help them narrow their request if they need it.

My goal here is to give askers a bit more confidence around the types of requests they might make, and to lightly guide them to be as specific in their requests as possible. Hopefully without being too wordy.

**Comment by mbbroberg** [comment]
@kaslin fantastic point. It's helpful to get people thinking about how we'll use this information so they can format accordingly. I'll push an update.

**Comment by mbbroberg** [comment]
@kaslin I'm offering a minimal addition. Wdyt? cc @rajula96reddy too

**Comment by rajula96reddy** [comment]
@mbbroberg The new template looks really cool. This makes it simpler and easier from a user perspective 👍

I do have a couple of asks.
- Should we keep the communication medium options open in the template? Say someone just wants a tweet, what do they do in that case?
- Also can we add another check that says 'my request is compliant with Kubernetes CoC'? Just to be safe.

**Comment by mbbroberg** [comment]
Looks like we need a different approver

**Comment by mbbroberg** [comment]
@parispittman so sneaky :smile:
